src/tools/wasm-interp.cc

```diff
@@ -1194,15 +1193,16 @@ static WabtResult on_register_command(Context* ctx,
       const WabtStringSlice* module_name = &ctx->env.modules.data[i].name;
       if (!wabt_string_slice_is_empty(module_name) &&
           wabt_string_slices_are_equal(&name, module_name)) {
-        module_index = (int)i;
+        module_index = static_cast<int>(i);
```

Comment:
This doesn't need to be fixed, but I'm curious why you are using "int" which can have very different semantics on different platforms. I would have at least expected type "size_t".